ripple/src/main/webapp/businessdetails.js

```diff
@@ -63,7 +63,11 @@ function bdOnload() {
   // Access 'businessDetails' collection
   var lambda = (doc) => {
         if (doc.exists) {
-          // TODO: Read galleryBlobKeys
```

Comment:
Add a brief comment

Reply:
similar to the other reads I have in this func

Reply:
fixed